rollup.config.mjs

```diff
@@ -1,6 +1,13 @@
 import fg from 'fast-glob';
 
-const inputFiles = fg.globSync(['lib/**/*.mjs', '!**/__tests__/**', '!lib/testUtils/**']);
+const inputFiles = fg.globSync([
+	'lib/**/*.mjs',
+	'!**/__tests__/**',
+	'!lib/testUtils/**',
+
+	// NOTE: We cannot support CJS for `cli.mjs` since the `meow` dependency is pure ESM.
```

Comment:
There's an alternative approach: switching to yargs.
e.g. https://github.com/conventional-changelog/get-pkg-repo/issues/60#issuecomment-888615817

Reply:
Ya, I still have the same concern. Can you open an issue about that to share the `meow` CJS problem?

Reply:
Okay, let's revisit it if a problem happens.